server/index.js

```diff
@@ -33,4 +31,9 @@ app.use(vueMiddleware({
 	config,
 }));
 
+if (config.app.enableSentry) {
+	app.use(Raven.requestHandler());
+	app.use(Raven.errorHandler());
```

Comment:
I think in this case we should only move the errorHandler and test to see if it makes a difference in sending errors automatically.

In our app, both our request and error handling are actually implemented inside the vue-middleware.js file. Theoretically, we would actually need them both declared before that. However we aren't using the standard express onError directly (it may be used in the vue bundle renderer) so moving the errorHandler may have no change.

As a test to see if changing the errorHandler placement has an effect, you could comment out the Manual statement we added in vue middleware and then trigger the server side error to see if it gets sent without us manually capturing the exception.

Reply:
Created ENG-2078 to revisit global error handling and logging in the node app.